build.gradle

```diff
@@ -20,8 +20,14 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven { url 'https://maven.google.com' }
-        google()
```

Comment:
Why?

Reply:
Used for maps interface

Reply:
Not so sure these are not the same thing. @unstablebrainiac any thoughts on this? The last thing we want is a messed up gradle config.

Reply:
This is unnecessay. Why do you even need JitPack? Have you tried compiling without these changes? If yes, did you face a problem? If yes, I'm sure we can come up with a better solution.

Reply:
jitpack is required for syncing TagView and circleIndicator directly from github. Without jitpack, they won't sync.

Reply:
[Agreed](https://stackoverflow.com/questions/46467561/difference-between-google-and-maven-url-https-maven-google-com).

Reply:
So only google() is fine or nothing at all?

Reply:
Just don't change anything that doesn't **need** to be changed. Any refactor can be done separately.